src/zeta_function_R_interface.c

```diff
@@ -7,6 +7,11 @@
 
 #include <math.h>
 
+//#include <Rcpp.h>
```

Comment:
these lines are not present in the master branch so you might as well remove them here

Reply:
This line is deleted now.